src/msw/webview_ie.cpp

```diff
@@ -963,14 +964,14 @@ bool wxWebViewIE::IsElementVisible(wxCOMPtr<IHTMLElement> elm)
             if(SUCCEEDED(elm2->get_currentStyle(&style)))
             {
                 //Check if the object has the style display:none.
-                if((style->get_display(&tmp_bstr) != S_OK) || 
-                   (tmp_bstr != NULL && (wxCRT_StricmpW(tmp_bstr, L"none") == 0)))
+                if((style->get_display(display_bstr) != S_OK) || 
+                   ((BSTR)display_bstr != NULL && (wxCRT_StricmpW(display_bstr, L"none") == 0)))
```

Comment:
OTOH we could have implicit conversion to `unspecified_bool_type` (see e.g. `wx/sharedptr.h` if you haven't heard about this idiom before) to avoid ugly casts/comparisons with `NULL` like this.

Or, perhaps even better, simply have an `IsOK()` accessor.